libs/core-ui/src/lib/util/TelemetryEventName.ts

```diff
@@ -60,5 +60,18 @@ export enum TelemetryEventName {
   IndividualCausalSelectedDatapointUpdatedFromDropdown = "RAI_IndividualCausalSelectedDatapointUpdatedFromDropdown",
   CausalTreatmentPolicyTabClick = "RAI_CausalTreatmentPolicyTabClick",
   CausalTreatmentPolicyNewTreatmentFeatureSelected = "RAI_CausalTreatmentPolicyNewTreatmentFeatureSelected",
-  LocalCausalEffectsFetchError = "LocalCausalEffectsFetchError"
+  LocalCausalEffectsFetchError = "LocalCausalEffectsFetchError",
+  LocalCausalEffectsFetchSuccess = "LocalCausalEffectsFetchError",
```

Comment:
should this be `LocalCausalEffectsFetchSuccess `instead of `LocalCausalEffectsFetchError`?

Reply:
Great catch!!